### PR TITLE
Support Laravel 11

### PR DESCRIPTION
**Pull request by saulens22**
This pull request adds support for Laravel 11 and uses its methods to get table column information.

Should be released as v3.0 as it's a breaking change.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -8,13 +8,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [ "8.1","8.2" ]
+        php: [ "8.2","8.3" ]
         composer-dependency: [ prefer-stable, prefer-lowest ]
```
Comment (romanzipp) [inline .github/workflows/tests.yml:12]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -8,13 +8,13 @@ jobs:
...
         exclude:
-          - php: "8.1"
-            composer-dependency: prefer-lowest
           - php: "8.2"
             composer-dependency: prefer-lowest
```
Comment (romanzipp) [inline .github/workflows/tests.yml:15]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -8,13 +8,13 @@ jobs:
...
+          - php: "8.3"
+            composer-dependency: prefer-lowest
```
Comment (romanzipp) [inline .github/workflows/tests.yml:17]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -31,7 +31,7 @@ jobs:
           coverage: none
 
       - name: Install dependencies
-        run: composer update --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist --${{ matrix.composer-dependency }}
+        run: composer update --no-interaction --no-scripts --no-progress --prefer-dist --${{ matrix.composer-dependency }}
```
Comment (romanzipp) [inline .github/workflows/tests.yml:34]:
```suggestion
        run: composer update --no-interaction --no-scripts --no-progress --prefer-dist
```

**Review by romanzipp — CHANGES_REQUESTED** [review]
Looks good, thanks! The composer dependency matrix value can be safely removed at this point

**Comment by saulens22** [comment]
All changes committed, should be good now

**Review by romanzipp — APPROVED** [review]

**Pull request merged**